### DISCUSSION TITLE
bounding box for unloaded payloads?

### DISCUSSION THREAD

**Discussion by sharktacos** [Autodesk/maya-usd#3330] [Q&A] [answered]
I'm trying to get bounding boxes to display for unloaded payloads using the extents hints attribute, following the example from the [USD Cookbook](https://github.com/ColinKennedy/USD-Cookbook/blob/33eac067a0a62578934105b19a2b9d8e4ea0646c/references/extents_hint_and_bounding_boxes/README.md). I'm using the all_in_one.usda file to test this:

````
#usda 1.0

def Cube "bounding_box_here" (
    kind = "component"
    payload = @something.usda@
)
{
    float3[] extentsHint = [(-1, -1, -1), (1, 2.5, 1)]
}
````
This works as advertised in USDview. It does not work in Maya, and unloading a payload shows nothing.

What I'm after is the behavior of Houdini which displays bounding boxes for unloaded payloads. 

Is this currently possible in Maya?

**Comment by sepux** [comment]
Not sure vanilla Maya USD but you can definitely do it through Bifrost USD

**Comment by jcoelho88** [comment, marked as answer]
@sharktacos  I think this would be very helpful.

I've tried myself with no luck and have seen others talk about generating a cube using the extents you want to represent the payload and I'd much rather see the behavior we get in USDView and Houdini. 

Cheers,

Jason Coelho

**Reply by sharktacos** [reply]
@jcoelho88 Thanks for confirming. Looks like this is a feature request then.
